pkg/persistence/events_test.go

```diff
@@ -65,16 +63,21 @@ func Test_SaveEvent(t *testing.T) {
 
 	event := Event{
 		Identifier: OTKGenerated,
-		Originator: token1,
+		Originator: "ONApi",
 		Count:      1,
 		DeviceType: Server,
 		Date:       time.Now(),
 	}
 
+	mock.ExpectBegin()
 	setupSaveEventMock(mock, event)
 
-	saveEvent(db, event)
+	event.Originator = token1
```

Comment:
it looks like this line is reverting the code on line 66, was there a specific reason for the change on 66?

Reply:
So I don't really know what caused the change to break this test. 

But this is what is happening: 

`token1` gets translated to `"ONApi"` in the saveEvent method so the expectation created in setupSaveEventMock needs the translated version of the token. Then I have to set it back to token1 so that the method can do the translation. 

If I revert  back the test fails and you get the following error: 

_Expected nil, but got: &errors.errorString{s:"ExecQuery '\n\t\tINSERT INTO events\n\t\t(source, identifier, device_type, date, count)\n\t\tVALUES (?, ?, ?, ?, ?) ON DUPLICATE KEY UPDATE count = count + ?', arguments do not match: argument 0 expected [string - aaaaaaaaaaaaaaaaaaaa] does not match actual [string - ONApi]"}_